frontend/.env.development

```diff
@@ -1,3 +1,3 @@
 VUE_APP_FRONTEND=http://127.0.0.1:8080
 VUE_APP_FUNCTION_KEY=SteUyT0o2YSt4JW8k2qZ5ShTBcQYQj1pxE8cO7oTrYI4ps92/53o9w==
-VUE_APP_BACKEND=https://www.theurlist.com
+VUE_APP_BACKEND=https://localhost:5001

```

Comment:
It depends on how you run the backend locally. It could change the port if run from IISExpress, VSCode or command line. It would be nice to add the steps to set this value of VUE_APP_BACKEND in the README.md.

Reply:
Done.